packages/loading-scripts/src/index.ts

```diff
@@ -54,21 +54,29 @@ if (!disableDynamicImports) {
   }
 }
 
-const rudderAnalyticsMount = () => {
+(window as any).rudderAnalyticsMount = () => {
   const rudderAnalyticsScript = document.createElement('script');
   rudderAnalyticsScript.src = `${sdkBaseUrl}/${sdkBuildType}/${sdkName}`;
   rudderAnalyticsScript.async = asyncScript;
-  document.head.appendChild(rudderAnalyticsScript);
+  if (document.head) {
+    document.head.appendChild(rudderAnalyticsScript);
+  } else {
+    document.body.appendChild(rudderAnalyticsScript);
+  }
 };
 
 if (typeof Promise === 'undefined') {
   const rudderAnalyticsPromisesScript = document.createElement('script');
   rudderAnalyticsPromisesScript.src =
-    'https://polyfill.io/v3/polyfill.min.js?features=globalThis%2CPromise&callback=rudderAnalyticsMount';
+    'https://polyfill.io/v3/polyfill.min.js?features=globalThis%2CPromise&callback=window.rudderAnalyticsMount';
```

Comment:
here you just need the name of the method, polyfil io assumes you have exposed it in window, revent this line please

Reply:
Done